### ISSUE TITLE
http://face4j.org link is empty

### ISSUE THREAD

**Issue by rossdm** [Codigami/face4j#11]
The http://face4j.org link provided on the Github page has no content. Perhaps the site's still in development?

**Comment by nischal** [comment]
Yeah, eventual goal is to have examples and "getting started" pages on it. For now, concentrating more on developing the API so that all the graph api related stuff is in there.
